docker/docker-compose.2004.57.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: vscode-swift:20.04-5.7
+    build:
+      args:
+        base_image: "swiftlang/swift:nightly-5.7-focal"
```

Comment:
do we want to use the release one now?

Reply:
Yes, just realised we are using the 5.6 nightly as well.  Will fix both